File: C/arnoob16_largestElementInTheArray.c
#include <stdio.h>
int main() {

    // Input the size of array
    printf("Enter the size of the array\n");
    int size; 
    scanf("%d", &size);

    // Declare an array
    int array[size];

    //Array Input
    printf("Enter the array element separated by spaces\n");
    for(int i=0; i<size;i++){
        //Input 1 element
        scanf("%d", &array[i]);
    }

    //Assuming 1st element is max
    int max = array[0];

    for(int i=0;i<size;i++){
        //Compare the element with the previous max element and assign it to max if a greater value is found.
        if(array[i] > max){
            max = array[i];
        }
    }
    //Print the max value with a suitable message.
    printf("The maximum element of the given array is : %d", max);
	return 0;
}

File: Java/arnoob16_largestElementInTheArray.java
import java.util.*;
public class arnoob16_largestElementInTheArray{
    public static void main(String[] args){
        // Made a Scanner Object
        Scanner in = new Scanner(System.in);

        // Input the size of array
        System.out.println("Enter the size of the array");
        int n = in.nextInt();

        // Declare an array
        int ar[] = new int[n];

        //Assign the min value possible to max variable
        int max = Integer.MIN_VALUE;

        //Array Input
        System.out.println("Enter the array element separated by spaces");
        for(int i=0;i<n;i++){
            //Input 1 element
            ar[i] = in.nextInt();

            //Compare the element with the previous max element and assign it to max if a greater value is found.
            if(max < ar[i])
                max = ar[i];
        }
        //Print the max value with a suitable message.
        System.out.println("The maximum element of the given array is : " + max);

        //Closing the scanner object to avoid un-necessary errors in IDE 
        in.close();
    }
}